pkg/controller/broadcastjob/broadcastjob_controller.go

```diff
@@ -53,9 +53,17 @@ func init() {
 	flag.IntVar(&concurrentReconciles, "broadcastjob-workers", concurrentReconciles, "Max concurrent workers for BroadCastJob controller.")
 }
 
+const (
+	jobNameLabelKey       = "job-name"
+	controllerUIDLabelKey = "controller-uid"
+)
+
 var (
 	concurrentReconciles = 3
 	controllerKind       = appsv1alpha1.SchemeGroupVersion.WithKind("BroadcastJob")
+
+	// AttachedPodLabelKeys gathers pod label keys for broadcastJob selector
+	AttachedPodLabelKeys = []string{jobNameLabelKey, controllerUIDLabelKey}
```

Comment:
Since we only have two constants，the array may not needed. Without the array, the code in webhook will not be complicated significantly.

Reply:
OK